modules/investigating-master-node-installation-issues.adoc

```diff
@@ -0,0 +1,197 @@
+// Module included in the following assemblies:
+//
+// *support/troubleshooting-installations.adoc
+
+[id="investigating-master-node-installation-issues_{context}"]
+= Investigating master node installation issues
+
+If you experience master node installation issues, determine master node, software defined network (SDN) and network operator status. Collect `kubelet.service`, `crio.service` journald unit logs and master node container logs for visibility into master node agent, CRI-O container runtime and Pod activity.
```

Comment:
Operator

Reply:
I have checked the entire PR and have now capitalised 'Operator', except within commands and variables.